Core/Src/state_machine.c

```diff
@@ -139,10 +119,9 @@ static int transition_functional_state(func_state_t new_state)
 		// DO NOT CHANGE WITHOUT CHECKING the bms fault timer, as if BMS timer is later could result in a fault/unfault/fault flicker.
 		osTimerStart(fault_timer, 5000);
 		HAL_IWDG_Refresh(&hiwdg);
-		transition_drive_state(OFF);
 		cerberus_state.nero =
 			(nero_state_t){ .nero_index = OFF, .home_mode = true };
```

Comment:
can you make home mode be false here so we go to the off screen which should theoretically be displaying the fault